cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/StreamUtils.java

```diff
@@ -425,12 +427,24 @@ public Long getResult() {
   }
 
   /**
-   * Gets a table name for stream consumer state stores within a given namespace.
+   * Gets a TableId for stream consumer state stores within a given namespace.
+   * @param namespace the namespace for which the table is for.
+   * @return constructed TableId
+   */
+  public static TableId getStateStoreTableId(Id.Namespace namespace) {
+    String tableName = String.format("%s.%s.state.store",
+                                     Constants.SYSTEM_NAMESPACE, QueueConstants.QueueType.STREAM.toString());
+    return TableId.from(namespace.getId(), tableName);
```

Comment:
Why the `TableId.from` doesn't take `Id.Namespace` (or at least have a overloaded method).